database/src/main/resources/db/migration/V1_69__finnish_ports.sql

```diff
@@ -0,0 +1,97 @@
+CREATE TABLE finnish_ports (
+ id SERIAL PRIMARY KEY,
+ code TEXT,
+ name TEXT,
+ location GEOMETRY,
+ created timestamp with time zone DEFAULT NOW(),
+ "created-by" TEXT REFERENCES "user" (id)
```

Comment:
Add one column to rule them all.
We need to know if column is added by user and approved by admin.
Not sure when those admins will approve anything, but it won't hurt if we have database support for it already.

I would suggest that all those ports that are from valid source are indicated something like
official_finnish_port = true
and all unvalidated custom ports are indicated something like
validated = false.
So the table could be like
id SERIAL PRIMARY KEY,
code TEXT,
name TEXT,
location GEOMETRY,
created timestamp with time zone DEFAULT NOW(),
"created-by" TEXT REFERENCES "user" (id)
official_finnish_port BOOL,
validated BOOL

tms.

Reply:
I don't want to add columns prematurely, we can already determine initial data because it has no creator.

Will ALTER TABLE when we get new requirements.